zeppelin-web-angular/src/app/share/header/header.component.html

```diff
@@ -67,7 +67,7 @@
           <li nz-menu-item (click)="logout()">Logout</li>
         </ng-container>
         <li nz-menu-divider></li>
-        <li nz-menu-item><a href="/">Old Version</a></li>
+        <li nz-menu-item><a href="/classic">Classic Version</a></li>
```

Comment:
should we use the same words? I suggest using either
1. "Classic UI" vs. "Default UI" or
2. "Switch to Classic UI" vs. "Switch to Default UI"

Reply:
I prefer a more specific phrasing, so I'll try 'Switch to Classic UI' vs. 'Switch to Default UI'.